scripts/delay_filter_run.py

```diff
@@ -12,24 +12,24 @@
 a = parser.parse_args()
```

Comment:
I think you meant to make this `ap = parser.parse_args()`, otherwise you'll get errors below because `ap` is not defined.

Reply:
thanks for catching this.